gcp_variant_transforms/pipeline_common.py

```diff
@@ -151,7 +149,8 @@ def _get_file_names(input_file):
   if not filesystems.FileSystems.exists(input_file):
     raise ValueError('Input file {} doesn\'t exist'.format(input_file))
   with filesystems.FileSystems.open(input_file) as f:
-    contents = map(str.strip, f.readlines())
+    contents = list(map(str.strip,
+                        [line.decode('utf-8') for line in f.readlines()]))
```

Comment:
I thought I left a comment about this in last review!
Anyways... why `contents` type was map and now is list?

Reply:
You did and I replied :D (make sure to expand all the comments in the conversations).
I said it was pylint, but that is wrong - I just retested without list and the actual reason for my change was to fix unit tests.

```
ERROR: test_validation_failure_for_wrong_pattern_in_input_file (gcp_variant_transforms.pipeline_common_test.PipelineCommonWithFileTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".../gcp-variant-transforms/gcp_variant_transforms/pipeline_common_test.py", line 224, in test_validation_failure_for_wrong_pattern_in_input_file
    input_pattern=None, input_file=filename)
  File ".../gcp-variant-transforms/gcp_variant_transforms/pipeline_common.py", line 94, in _get_all_patterns
    matches = filesystems.FileSystems.match(patterns, [1] * len(patterns))
TypeError: object of type 'map' has no len()
```

Apparently map doesn't have len anymore, and that's what we use in our unit tests for verification.